communities/epfl-hackers/community-impact-proposal.md

```diff
@@ -0,0 +1,76 @@
+## Summary of the Community's Needs
+
+- **Community demographics**: Currently a very small community of about 5 people, we aim to grow to  150 people during the next academic year. Our community will include anyone with a passion for tech and an interest in working on passion projects outside of academia, regardless of their educational background. Currently the community doesn't have many resources available, except for LauzHack which was organized with the aim of kick starting the community.
+
+
+- **Communication**: At the moment communications are direct messages between members by email/IM, but as the community grows a Facebook group and Slack channel will be introduced as required.
+
+- **Culture**: At the moment, we don't have any meetups or regular group activity so it has been hard to develop a community culture. People pretty much work on their own individual projects and interactions are limited to asking for help if and when needed.
+
+## Your Proposal
+
+Launching a new student association at EPFL for the community, and organizing regular meet-ups for the members of the association.
+
+- **Timing**: For us the key timing will be the start of the next semester. We would ideally like to setup an official university association by the end of the summer, thereby enabling us to pitch the associations at the associations fair at the start of the academic year. This will be key to letting people (especially first years) know that we exist and how to get in touch with us. Soon after this we would start weekly/monthly meetups for the community to bond and get comfortable with other members. The idea behind these meetups is to give community members the chance to learn something cool and new, and also be able to discuss any issues they are dealing with. Such meetups will also be ideal for promoting hackathons and encouraging traveling to hackathons all over Europe.
+
+- **Community demographics**: We definitely want to increase the size of the community many fold. To do so the first step is to become an official university association. Following this, our main outreach would be through word-of-mouth as people who enjoy being a part of the community will hopefully tell their friends about it. While this is a great form of 'organic growth', the community might lack diversity as theres a high probability that most people who join us will only be from a computer science background. In order to attract people with more diverse backgrounds we will reach out to other departments in the university (not just Computer Science).
```

Comment:
Are there any affinity groups in the university or beyond where you might also be able to alert diverse students to your events?